coffee/chosen.jquery.coffee

```diff
@@ -34,6 +34,7 @@ class Chosen extends AbstractChosen
     container_props =
       'class': container_classes.join ' '
       'title': @form_field.title
+      'tabIndex': '-1'
```

Comment:
When attempting to click on an item in the Chosen drop but missing by a pixel or two, we don’t want the Chosen drop to close. We can add a tab index, which allows it to receive focus — resulting in the following: 

0. The search input has focus, and the user attempts to use their mouse to click on a result.
1. The user begins clicking. The search input blurs, triggering a `focusout` event, which is caught on the container and a `setTimeout` is scheduled. 
2. The container receives the mouse event, which focuses the container. The container triggers a `focusin` event, which is caught on the container and nothing happens (`@active_field` is true).
3. The `setTimeout` expires, checking to see if the container contains `document.activeElement`, which is in this case, itself. As [defined in the spec](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains), a Node `.contains()` itself, and therefore the drop is not closed.